packages/tailwindcss/src/index.ts

```diff
@@ -314,12 +330,13 @@ async function parseCss(
 
 export async function compile(
   css: string,
+  base: string,
```

Comment:
I think I'd move this to `CompileOptions`. While a correct value is required for `@import` statements to work _on the filesystem_, that's not necessarily the case for Play or the CDN given that they'll need to overload `loadModule` and `loadStylesheet`. I think either `/` or `./` is a reasonable "if you don't provide it" default.

Reply:
Yup agree but I was already knee-deep in the migration to the second arg when I realized this is maybe not a good idea. Will change!